Tests/LibShell/Tests/test-commons.inc

```diff
@@ -0,0 +1,14 @@
+fail()
+{
+#The AST ends up at stderr, and CI checks for 'fail'(uppercase)
+#Trick it into not failing the test.
+    echo "FA"
+         "IL:" $*
```

Comment:
Ah, that's because you somehow reformatted this file.
Please don't, this isn't even correct :)

Reply:
Done :)